apps/hubble/src/addon/src/store/relationship_store.rs

```diff
@@ -496,14 +496,19 @@ impl RelationshipStore {
         Ok(promise)
     }
 
-    pub fn get_relationships_by_source(
+    pub fn get_relationships_by_related_object_ref(
         store: &Store,
-        source: &ObjectRef,
+        related_object_ref: &ObjectRef,
+        related_object_type: i32,
         name: String,
         page_options: &PageOptions,
     ) -> Result<MessagesPage, HubError> {
-        let prefix = RelationshipStoreDef::make_relationship_by_source_key(source, 0, None);
-
+        let prefix = if related_object_type == RelatedObjectTypes::Source as i32 {
```

Comment:
I see, so the related_object_type refers to what the object ref is relative to the relationships being queried. At first thought related_object_type was talking about the type of the object which was confusing.

Reply:
I agree that it's confusing. There were already 2 other params in the mix referring to a type of something:
- type of the relationship
- type of object reference (FID or object ref)
So this third type is referring to whether the object ref serves as the source or target for the relationship.
Names are hard - I'm happy to take suggestions.

Reply:
Perhaps something with the word "direction"?